security/middleware.py

```diff
@@ -340,20 +343,24 @@ class ContentNoSniff(MiddlewareMixin):
 
     def __init__(self, get_response=None):
         super().__init__(get_response)
-        warnings.warn((
-            'DEPRECATED: The middleware "{name}" will no longer be '
-            'supported in future releases of this library. Refer to {url} for '
-            'an alternative approach with regards to the settings: {settings}'
-        ).format(
-            name=self.__class__.__name__,
-            url=DJANGO_SECURITY_MIDDLEWARE_URL,
-            settings="SECURE_CONTENT_TYPE_NOSNIFF"))
+        warnings.warn(
```

Comment:
Let's drop this deprecated middleware as well

Reply:
Updated - https://github.com/sdelements/django-security/pull/97/commits/a3af3b829cc03630cc8cf49cdca32c827c1a2f7c